src/views/About/index.jsx

```diff
@@ -101,7 +100,7 @@ const About = () => {
         <Typography variant="subtitle1">
           Found a bug?
           <a href="mailto:cts@gordon.edu?Subject=Gordon 360 Bug">
-            <Button style={{ color: gordonColors.primary.cyan }}>Report to CTS</Button>
+            <Button style={{ color: 'var(--mui-palette-primary-main)' }}>Report to CTS</Button>
```

Comment:
Two things here:
1. You appear to have switched the color of this `Button` from cyan to blue.
2. You shouldn't need to pass this as an inline style, you can just pass the `color` prop to the button.

Reply:
Good point, I've made that change :+1: